### DISCUSSION TITLE
Please give me some stable UI tools

### DISCUSSION THREAD

**Discussion by thienphuoc** [axmolengine/axmol#1150] [Q&A] [answered]
Hi all, 

before, I used the cocos studio for UI Design. But now, It deprecated right? For Axmol, Please give me some stable tools for building UI . 

Thanks

**Comment by rh101** [comment, marked as answer]
As far as I know, you have 2 options:

1 - https://www.fairygui.com/

2 - Write your own UI builder

**Comment by aismann** [comment]
> before, I used the cocos studio for UI Design. But now, It deprecated right? 
> 
@halx99 
As far as I know you be the developer of cocos studio. Is it deprecated?

**Reply by thienphuoc** [reply]
I played the examples of FairyGUI on Axmol[ as source here ](https://github.com/fairygui/FairyGUI-cocos2dx)

It very nice ! (y)

https://user-images.githubusercontent.com/17909394/229079726-cb7cab02-790b-4973-8959-07c15c00b215.mov

-------------------------------------------------------------

Stable project for fresher man: https://github.com/thienphuoc/FairyGUI-axmol.git

**Reply by aismann** [reply]
> I played the examples of FairyGUI on Axmol[ as source here ](https://github.com/fairygui/FairyGUI-cocos2dx)
> 
> It very nice ! (y)

@thienphuoc 
Thanks about v-clip, we known it ;)
Can you mark as answer?

**Reply by thienphuoc** [reply]
Axmol project for FGUI :  https://github.com/thienphuoc/FairyGUI-axmol.git 😅👍
